stopdnsrebind.go

```diff
@@ -39,7 +39,7 @@ func (a Stopdnsrebind) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dn
 		}
 	}
 
```

Comment:
Need to write the unaltered message from the non-writer, with the responsewriter before returning ... `w.WriteMsg(nw.Msg)`